web_access_rule_buttons/models/models.py

```diff
@@ -22,19 +22,16 @@ def check_access_rule_all(self, operations=None):
             operations = ["read", "create", "write", "unlink"]
         result = {}
         for operation in operations:
-            try:
-                self.check_access_rule(operation)
-            except exceptions.AccessError:
-                result[operation] = False
-            if (
-                self.is_transient()
-                or self.ids
-                and self.env.user.has_group("base.user_admin")
-            ):
+            if self.is_transient() or not self.ids:
```

Comment:
i feel like changing the logic of another module deserves more attention than just a commit message that says fixed tests.

unfortunately i cannot just remove my approval for just the test fix commit: https://stackoverflow.com/a/44747392/925125

Do you know where the ignoring AccessErrors for `self.ids and self.env.user.has_group("base.user_admin")` came from (what the intention was)? (As this logic was removed in the fix tests commit)

Reply:
Well, in my opinion it would have been better to explain that in the commit message, as those will be forever with the commit (unless someone uses rebase commits).

And if we start accepting commits within the OCA that copy contributions of other contributors without crediting them we are the same as Odoo but i guess that's how we are now.

I approved the initial commits but not the uncredited use of other peoples work.

Reply:
From my POV this is a small change that does not really imply "stealing work" from someone else. 
Yet, in both work, there's no way to reuse the same commit to address the same problem.

Because it has been poorly named and not separated.
The change is not to fix tests, is to fix the code and then adapt tests.
Also, we'll have a conflict when trying to port this changes fwd unless the author of that PR does not cherry-pick this commit in his PR and discard his changes on this file.